__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+afterAll(() => {
+  return db.end();
+});
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("/api/topics", () => {
+  test("GET 200: Responds with an array with all the topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(typeof topic.description).toBe("string");
+          expect(typeof topic.slug).toBe("string");
+        });
+      });
+  });
+  test("GET 404: Responds with an error when path used does not exist", () => {
```

Comment:
would advise moving this 404 path not found test to its own describe block now - it will cover any route not found so not linked to /api/topics per se. 👍👍 for well considered test descriptions!